client/src/modules/App.jsx

```diff
@@ -2,13 +2,56 @@ import React from 'react';
 import Nav from './Nav';
 import Home from './Home';
 
-export default React.createClass({
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loggedIn: '',
+    };
+
+    this.authenticateUser = this.authenticateUser.bind(this);
+  }
+
+  authenticateUser() {
+    var context = this;
+
+    fetch('http://127.0.0.1:3000/api/users/authenticate', {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+    })
+    .then(function(response) {
```

Comment:
Heads up, when we start actually using the ES6 linter, it'll want the ES6 version of all of these.

E.g. response => response.json()